lib/ts/recipe/thirdpartyemailpassword/components/themes/translations.ts

```diff
@@ -0,0 +1,12 @@
+import { defaultTranslationsEmailPassword } from "../../../emailpassword/components/themes/translations";
+import { defaultTranslationsThirdParty } from "../../../thirdparty/components/themes/translations";
+
+export const defaultTranslationsThirdPartyEmailPassword = {
+    en: {
+        ...defaultTranslationsThirdParty.en,
+        ...defaultTranslationsEmailPassword.en,
+
+        THIRD_PARTY_EMAIL_PASSWORD_SIGN_IN_AND_UP_HEADER_TITLE: "Sign Up or Log In",
+        THIRD_PARTY_EMAIL_PASSWORD_SIGN_IN_AND_UP_DIVIDER_OR: "or",
```

Comment:
You need a space around or?

Reply:
No, this doesn't have any text around it.

Reply:
okay!